app/main.ipynb

```diff
@@ -31,17 +31,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code for task 0 here"
+    "data_set = pd.read_csv(\"../data/adult.csv\")\n",
+    "data_set = data_set.replace(\"?\", np.nan)\n",
+    "data_set[\"salary K$\"] = np.where(data_set[\"salary K$\"] <= 0, np.nan, data_set[\"salary K$\"])\n",
+    "data_set = data_set.dropna()\n",
+    "\n",
+    "data_set[\"salary\"] = np.where(data_set[\"salary K$\"] <= 50, \"<=50K\", \">50K\")\n",
```

Comment:
The transformation of the 'salary' column assumes that 'salary K$' values less than or equal to 50 should be categorized as '<=50K'. Verify that this transformation is correct based on the dataset's context.